dashboard/app/assets/javascripts/levels/external.js

```diff
@@ -0,0 +1,2 @@
+//= require codemirror
+//= require_self
```

Comment:
I don't think the require_self is strictly needed unless there's javascript within this file that you want to included into the bundle. just fyi, there's no harm in leaving it as-is